apps/omg_watcher/lib/db/transaction.ex

```diff
@@ -78,6 +86,7 @@ defmodule OMG.Watcher.DB.Transaction do
       )
 
     Repo.all(query)
```

Comment:
Let's preload with query
```suggestion
    Repo.all(query, preload: :block)
```